src/scripts/librarybrowser.js

```diff
@@ -83,7 +83,7 @@ define(["userSettings"], function (userSettings) {
 
             if (html += '<div class="listPaging">', showControls) {
                 html += '<span style="vertical-align:middle;">';
-                html += (totalRecordCount ? startIndex + 1 : 0) + "-" + recordsEnd + " of " + totalRecordCount;
+                html += Globalize.translate("ListPaging").replace("{0}", (totalRecordCount ? startIndex + 1 : 0) + "-" + recordsEnd).replace("{1}", totalRecordCount);
```

Comment:
The translate library has a method to replace the variables.

Reply:
That's how is done in other parts of the code, so I used that way to keep that consistent. Will replace that then with the built-in method in every part where it's used across the entire web code. Any hint at the function and an example so I can accomplish this?

Thanks for pointing that out!

Btw, adding the strings at `en-US` should be enough for Weblate to populate it across all languages, right?

Reply:
The media type provider selectors implement this, I think you just have to overload the translate method with the extra strings and it will fill them in properly.